### DISCUSSION TITLE
Request-FalconToken : The script failed due to call depth overflow.

### DISCUSSION THREAD

**Discussion by IntuitiveCol** [CrowdStrike/psfalcon#142] [Q&A] [answered]
Hello, I am using the basic install script and am running into the error stated in the title. Below is a snip of the error.....

Request-FalconToken : The script failed due to call depth overflow.
+                 & $MyInvocation.MyCommand.Name
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (0:Int32) [Request-FalconToken], RuntimeException
    + FullyQualifiedErrorId : CallDepthOverflow,Request-FalconToken

When I try to follow the steps described in a previous discussion I run into an error stating that the clientID is reserved for future use when attempting the request-falcontoken portion...

The '<' operator is reserved for future use.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : RedirectionNotSupported

I have not altered the initial script in anyway, I believe the correct modules are installed as i can import it and run the PSFalcon specific commands. Any thoughts?

**Comment by bk-cs** [comment, marked as answer]
Which script are you using? The sensor install scripts do not require PSFalcon.

Based on the error, I'm guessing it's related to using the wrong cloud. By default, `Request-FalconToken` requests (along with the token requests built into most scripts that I write) point to US-1.

Is that the cloud your Falcon environment was created in?

**Comment by IntuitiveCol** [comment]
It was US-2 Which I had it pointed to, I have abandoned the powershell installer in favor of a tool we recently acquired. Thank you for your reply!
